sql-select/cheap-to-rent.sql

```diff
@@ -0,0 +1,5 @@
+select "filmId",
+      "title"
+from "films"
+order by "rentalRate" ASC
+limit 50
```

Comment:
This doesn't guarantee that the `"rentalRate"` will be less than `1` dollar to rent:

---

![image](https://user-images.githubusercontent.com/16143771/144165750-b83569f4-80d1-4317-8dc2-4b66f7306699.png)